### PR TITLE
[webview_flutter] Added 'allowsInlineMediaPlayback' property

### PR DESCRIPTION
**Pull request by mehmetf**
## Description

The PR includes following changes:
* adds `allowsInlineMediaPlayback` property to WebView to support inline media playback in iOS.
* switched to using collection literals for `Set`.

## Related Issues

[#25630](https://github.com/flutter/flutter/issues/25630)
[#42099](https://github.com/flutter/flutter/issues/42099)

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Comment by mehmetf** [comment]
@blasten This is ready to be reviewed.

File: packages/webview_flutter/lib/webview_flutter.dart [hunk 1]
```diff
@@ -333,6 +335,13 @@ class WebView extends StatefulWidget {
   ///     * When a navigationDelegate is set HTTP requests do not include the HTTP referer header.
   final NavigationDelegate? navigationDelegate;
 
+  /// Controls whether inline playback of HTML5 videos is allowed on iOS.
+  ///
+  /// This field is ignored on Android.
```
Comment (amirh) [inline packages/webview_flutter/lib/webview_flutter.dart:340]: nit: mention that inline playback is allowed on Android.

**Review by amirh — APPROVED** [review]
LGTM (modulo one small documentation nit)

Thanks for the extra test coverage!

**Comment by gokhanbe** [comment]
Although I made all the changes, the ipad is working in full screen but the iphone does not work

https://user-images.githubusercontent.com/57063848/104122686-8abf8700-5357-11eb-96b4-beb18cc5c4e1.mov


https://user-images.githubusercontent.com/57063848/104122691-901cd180-5357-11eb-9dcc-aa6b842263de.mov

**Pull request merged**